cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerUpgradeService.java

```diff
@@ -78,11 +90,24 @@ private BigDecimal determineUpgradeLogic(String stackProductVersion, ClustersRes
                     LOGGER.debug("Last upgrade command ({}) is not active, it was {} successful and {} retryable, submitting it now", upgradeCommandId,
                             commandSuccess ? "" : "not",
                             commandCanRetry ? "" : "not");
-                    upgradeCommandId = callUpgrade(stackProductVersion, clustersResourceApi, stack);
+                    upgradeCommandId = executeUpgrade(stackProductVersion, clustersResourceApi, stack, postRuntimeUpgrade, command);
```

Comment:
pls externalise `command` as a constant in this class instead of passing as parameter here.